oddbird/oddleventy
# spacing adjustment for search button in header

![](https://source.unsplash.com/featured/?cute,animal&Craghy5rME)

## Description
Quick Fix spacing of header area on small screens

## Steps to test/reproduce
Make your browser narrow and see the spacing between logo and search button (and light/dark).
Make sure the menu isn't getting pushed off the right side of the viewport.

## Show me
~380px wide:
![Screen Shot 2022-08-31 at 10 13 46 PM](https://user-images.githubusercontent.com/1581694/187824588-44c5b252-e6ee-4712-987d-0d8a733cbec6.jpg)

slightly wider:
![Screen Shot 2022-08-31 at 10 14 05 PM](https://user-images.githubusercontent.com/1581694/187824619-a716b1a6-b5f6-4978-b935-c4f49fa5fe0b.jpg)

wide:
![wide](https://user-images.githubusercontent.com/1581694/187824664-b3ed85cc-97dd-4c3c-b900-737df9847c61.jpg)


# REMEMBER: Attach this PR to the Trello card
https://trello.com/c/wXTp2JCn/188-quick-search-spacing-fix
